cmd/main.go

```diff
@@ -116,14 +120,33 @@ func main() {
 	}
 
 	// Base Bigtable cache
-	var baseTable *cbt.Table
+	var baseTables []*cbt.Table
 	if *useBaseBt {
-		// Base cache
-		baseTable, err = server.NewBtTable(ctx, *storeProject, baseBtInstance, *baseTableName)
-		if err != nil {
-			log.Fatalf("Failed to create BigTable client: %v", err)
+		if *useImportGroup {
+			var c bigtable.TableConfig
+			if err := json.Unmarshal([]byte(*importGroupTables), &c); err != nil {
+				log.Fatalf("Failed to load import group tables config")
+			}
+			tableNames := c.Tables
+			bigtable.SortTables(tableNames)
```

Comment:
Are you going to check that there are multiple of them?  Elsewhere we decide "import group mode" based on number of bigtables.

I wonder if for consistency we should propagate the "*useImportGroup" flag and use that instead?

Reply:
Done